auto_selfcal.py

```diff
@@ -78,6 +78,18 @@
 check_all_spws=False   # generate per-spw images to check phase transfer did not go poorly for narrow windows
 apply_to_target_ms=False # apply final selfcal solutions back to the input _target.ms files
 sort_targets_and_EBs=False
+run_findcont=True
```

Comment:
I wonder if we shouldn't make this slightly more complicated.
default for run_findcont should be False I think

1. check if cont.dat exists, if not, go to 3
2. if cont.dat exists, does it have all targets (might need a new routine to check this), if not go to 3 (backup old version first)
3. run findcont

Reply:
For 2., if cont.dat does exist and does have all the targets, would it then skip doing findcont? If so, we may want an additional parameter to override that decision (in case someone really wants to redo findcont - though that would beg the question of why the cont.dat file is there in the first place...).

If not, and we'd still want it to do findcont anyways, then I think this would simplify to just backing up cont.dat, if it exists?

Reply:
If cont.dat exists for all targets, I would opt for skipping. If the user wants to re-run, I think it could be better to have then need to remove their existing cont.dat.

Reply:
That sounds good to me. I'll add these conditions in. In the event that cont.dat exists and does have all targets in it, I will have it spit out a message and skip doing findcont.